src/utils/tasks.ts

```diff
@@ -50,3 +51,16 @@ export const getFlatArrayFromObjectValues = (survey: any): Task[] => {
   )
   return arrayFlatTasks
 }
+
+export const getUnansweredGuidelines = (survey: any, auditProgram: any): any => {
+  const answeredGuide = Object.values(survey.answers)
+  const totalGuidelines = Object.values(auditProgram.guidelines)
+  const pkOfAnsweredGuide = answeredGuide.map((answer: any) => answer.guideline_pk)
+
+  const unansGuideline = totalGuidelines.filter(
+    (guideline: any) => pkOfAnsweredGuide.indexOf(guideline.pk) === -1
+  )
```

Comment:
This could be called ```notAnsweredGuidelines``` or ```unansweredGuidelines```. We are looping an array of guidelines, so each item will be of type ```Guideline```